powerline-separators.el

```diff
@@ -79,8 +80,10 @@ RED, GREEN and BLUE should be between 0.0 and 1.0, inclusive."
 (defun pl/pattern (lst)
   "Turn LST into an infinite pattern."
   (when lst
-    (let ((pattern (cl-copy-list lst)))
-      (setcdr (last pattern) pattern))))
+    (let ((r (make-ring (length lst))))
```

Comment:
could we use `ring-convert-sequence-to-ring`?

Reply:
Didn't even know it existed lol